pkg/storage/memory/memory.go

```diff
@@ -327,6 +339,11 @@ func (s *MemoryBackend) read(ctx context.Context, store string, tk *openfgav1.Tu
 	return &staticIterator{records: matches}, nil
 }
 
+type TupleChangeRec struct {
```

Comment:
Does this need to be exported?

Reply:
No. Will update to private (still getting used to `golang` style)